mkdocs.yml

```diff
@@ -138,6 +138,7 @@ nav:
           - 'FIDO': 'admin/admin-ui/fido-menu.md'
           - 'SMTP': 'admin/admin-ui/smtp-menu.md'
           - 'Users': 'admin/admin-ui/userMgmt-menu.md'
+        - 'FAQ & Troubleshooting': 'admin/admin-ui/faq.md'
```

Comment:
Hi @duttarnab 

Changes at this line are fine But check line 132. 
It is reinserting the problematic configuration. Maybe this branch was created before we fixed the issue. Can merge changes from `main` or apply your changes on a fresh branch and create a new PR?

Reply:
done